run_iwyu_tests.py

```diff
@@ -139,6 +199,13 @@ def RunOneTest(self, filename):
     clang_flags = self._clang_flags_map.get(filename, None)
     if clang_flags:
       logging.info('%s: Using clang flags %s', filename, str(clang_flags))
+    include_flags = self._include_map.get(filename, None)
+    if include_flags:
+      logging.info('%s: Using include flags %s', filename, str(include_flags))
```

Comment:
I wanted to be explicit about the includes so one can easier see, where include paths were set.